README.md

```diff
@@ -3,59 +3,72 @@
 Provides custom Drupal console commands and chains.
 
 ## Requirements
-- Composer https://getcomposer.org/
-- Drupal console https://drupalconsole.com/
+
+- [Composer](https://getcomposer.org/download/)
+- [Drupal Console Launcher](https://hechoendrupal.gitbooks.io/drupal-console/content/en/getting/launcher.html)
 
 ## Installation
 
-curl -L https://goo.gl/DcpKPN | sh
+You may follow either the scripted or manual steps below:
 
-or install it manually:
+### Scripted:
 
-### Requirements:
- - Composer: https://getcomposer.org/download/
- - Launcher: https://hechoendrupal.gitbooks.io/drupal-console/content/en/getting/launcher.html
+1. Get and run our installation script with a single command:
 
-### Run the following:
-- Init Drupal
 ```
-drupal -n --override init
+curl -L https://goo.gl/DcpKPN | sh
 ```
 
-- Set the environment i.e.
-```
-drupal settings:set environment dev
-```
+### Manual:
 
-- Install the Drupal Extend plugin (https://github.com/hechoendrupal/drupal-console-extend)
-```cd ~/.console/
-composer create-project \
-drupal/console-extend ~/.console/extend \
---no-interaction
-```
+1. Init Drupal console
 
-- Remove example (optional)
-```
-cd ~/.console/extend;
-composer remove drupal/console-extend-example
-```
+	```
+	drupal -n --override init
+	```
 
-- Install Dennis Digital Commands
-```
-cd ~/.console/extend;
-composer require dennisdigital/drupal_console_commands:extend_properly-dev
-```
+1.  Set the environment
```

Comment:
Should the number be 2 here?

Reply:
I think Markdown handles the sequence correctly - https://github.com/dakala/drupal_console_commands/blob/b2b4b451f7c88838cfc7a5b45b070cd763c07cf6/README.md